runtime/windbg/module.xml

```diff
@@ -42,12 +42,12 @@
 			<include path="j9oti"/>
 			<include path="j9dbgext"/>
 			<include path="j9jit_vm"/>
-			<include path="../tr.source/codegen" type="relativepath"/>
-			<include path="../tr.source/base" type="relativepath"/>
-			<include path="../tr.source/codegen/sched" type="relativepath"/>
-			<include path="../tr.source/optimizer" type="relativepath"/>
+			<include path="../compiler/codegen" type="relativepath"/>
+			<include path="../compiler/base" type="relativepath"/>
+			<include path="../compiler/codegen/sched" type="relativepath"/>
+			<include path="../compiler/optimizer" type="relativepath"/>

```

Comment:
The above 4 paths no longer exists.  Delete.

Reply:
Fixed in 626683a